core-bundle/contao/drivers/DC_Table.php

```diff
@@ -2209,16 +2195,38 @@ public function edit($intId=null, $ajaxId=null)
 		// Versions overview
 		if (($GLOBALS['TL_DCA'][$this->strTable]['config']['enableVersioning'] ?? null) && !($GLOBALS['TL_DCA'][$this->strTable]['config']['hideVersionMenu'] ?? null))
 		{
-			$version = $objVersions->renderDropdown();
+			$parameters['version_dropdown'] = $objVersions->renderDropdown();
 		}
 		else
 		{
-			$version = '';
+			$parameters['version_dropdown'] = '';
 		}
 
+		// Messages
+		$parameters['message'] = Message::generate();
```

Comment:
I think this is too early. If any messages happen to be added of the edit buttons, they would be missing. While it is unlikely you would add any messages there, we should adhere to the original execution order as much as possible (for the purpose of these PRs).

Reply:
Changed in 1e62ae7f919d91eef740afc27cb36033092ba041. But honestly, with this argumentation we should rather render messages last IMHO. But let's not have this discussion here.

Reply:
So I would move it down, I think. Do you agree?

Reply:
Yes 👍

Reply:
Done in c7da071d6cf7879c0746bf311eb08cb979c65834. I also grouped the form buttons (liked your naming) into the other form parameters. Now the logical things are also close to each other.